js/core/component.js

```diff
@@ -160,7 +122,7 @@ var Component = Class.inherit({
             this._setOptionsByReference();
             this._setDeprecatedOptions();
             this._options = this._getDefaultOptions();
```

Comment:
Why we keep `this._options` property here?

Reply:
We cannot just get rid of this field. I'll do it in next PR